sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -2898,4 +2899,33 @@ class HiveDDLSuite
       }
     }
   }
+
+  test("SPARK-33865: Hive DDL with avro should check col name") {
+    withTable("tbl", "t1") {
+      withView("view1") {
+        spark.sql("CREATE TABLE tbl(id long)")
+        spark.sql("INSERT OVERWRITE TABLE tbl VALUES 4")
+        spark.sql("CREATE VIEW view1 AS SELECT id FROM tbl")
```

Comment:
nit: we can prepare the data by `spark.range(1).createTempView("v")`, instead of creating a table then a view.

Reply:
> nit: we can prepare the data by `spark.range(1).createTempView("v")`, instead of creating a table then a view.

Updated